modules/oakcrime/terraform.tf

```diff
@@ -28,21 +25,61 @@ resource "aws_instance" "oakcrime" {
       "sudo -u tdooner chmod 600 ~tdooner/.ssh/authorized_keys",
       "sudo usermod -aG sudo tdooner && sudo passwd -de tdooner",
       "echo 'tdooner ALL=(ALL) NOPASSWD:ALL' | sudo tee '/etc/sudoers.d/tdooner' && sudo chmod 440 /etc/sudoers.d/tdooner",
-
       "sudo useradd --create-home --shell /bin/bash rik",
       "sudo -u rik mkdir -p ~rik/.ssh",
       "sudo -u rik bash -c 'echo \"${file("ssh-keys/rik.pub")}\" > ~rik/.ssh/authorized_keys'",
       "sudo -u rik chmod 600 ~rik/.ssh/authorized_keys",
       "sudo usermod -aG sudo rik && sudo passwd -de rik",
-      "echo 'rik ALL=(ALL) NOPASSWD:ALL' | sudo tee '/etc/sudoers.d/rik' && sudo chmod 440 /etc/sudoers.d/rik"
+      "echo 'rik ALL=(ALL) NOPASSWD:ALL' | sudo tee '/etc/sudoers.d/rik' && sudo chmod 440 /etc/sudoers.d/rik",
     ]
   }
 }
 
 resource "aws_route53_record" "oakcrime" {
   zone_id = "${var.zone_id}"
-  name = "oakcrime"
-  type = "A"
-  ttl = 60
+  name    = "oakcrime"
+  type    = "A"
+  ttl     = 60
   records = ["${aws_instance.oakcrime.public_ip}"]
 }
+
+#########################
+# Beanstalk applications
+#########################
+
+module "ci_user" {
+  source = "github.com/openoakland/terraform-modules//ci_user?ref=ci-user"
+
+  ci_username = "oakcrime-ci"
+}
+
+module "app_oakcrime" {
+  source = "github.com/openoakland/terraform-modules//beanstalk_app?ref=postgresdb"
```

Comment:
I assume the plan is to eventually go back and update the "ref=" part to be a version number rather than the branch, once the branch is merged, right?

Reply:
Yes, exactly. Open to better ways to handle this one-two-one dance.